x-pack/plugin/core/src/main/resources/org/elasticsearch/xpack/entsearch/analytics/behavioral_analytics-events-template.json

```diff
@@ -1,5 +1,5 @@
 {
-  "index_patterns": ["behavioral_analytics-events-*"],
+  "index_patterns": ["${event_data_stream.index_pattern}"],
```

Comment:
ℹ️ Now using a variable replaced with the class constant for the index pattern.
It will be easier to maintain this way.